lib/modelMaterialsCommon.js

```diff
@@ -727,6 +729,85 @@ function lightDefaults(gltf) {
     }
 }
 
+function getShaderVariable(accessor) {
+  var type = accessor.type;
+  if (type === 'SCALAR') {
+    return 'float';
+  }
+  return type.toLowerCase();
+}
+
+function ensureSemanticExistenceForPrimitive(gltf, primitive) {
+  var accessors = gltf.accessors;
+  var materials = gltf.materials;
+  var techniques = gltf.techniques;
+  var programs = gltf.programs;
+  var shaders = gltf.shaders;
+
+  var attributes = primitive.attributes;
+  var materialId = primitive.material;
+  var material = materials[materialId];
+  var techniqueId;
+  var technique;
+  if (defined(material)) {
+    techniqueId = material.technique;
+    technique = techniques[techniqueId];
+  }
+  var programId;
+  var program;
+  if (defined(technique)) {
+    programId = technique.program;
+    program = programs[programId];
+  }
+  var vertexShaderId;
+  var vertexShader;
+  if (defined(program)) {
+    vertexShaderId = program.vertexShader;
+    vertexShader = shaders[vertexShaderId];
+  }
+
+  for (var semantic in attributes) {
+    if (attributes.hasOwnProperty(semantic)) {
+      if (!defined(techniqueParameterForSemantic(technique, semantic))) {
+        if (defined(technique)) {
+          var accessorId = attributes[semantic];
+          var accessor = accessors[accessorId];
+          var lowerCase = semantic.toLowerCase();
+          var attributeName = 'a_' + lowerCase;
```

Comment:
So I don't think this matters. We replace the `program.attribute`, `technique.attribute`, `technique.parameter` and in the shader with the same case, so Cesium should handle it correctly because of the semantic.